remote-config.jsonc

```diff
@@ -123,6 +123,15 @@
                 },
                 {
                     "message": "Did you know you can set environment variables with the `.ddev/.env` file?"
+                },
+				{
+                    "message": "Has one of your containers suddenly crashed? Is `ddev describe` returning a black file? Do not worry, type `ddev logs --follow --time` for insights into such issue"
```

Comment:
A _blank_ file?

Reply:
I’d also try to get this shorter for brevity. We generally “run” commands instead of “type” them. Don’t forget ending punctuation.

Reply:
Good point. Maybe a blank table is more descriptive. 
If the ddev web server keeps crashing one only gets an empty table.

Reply:
Probably just shorten it in general rather than trying to specify a specific thing.